test_e2e/dest/onedrive.sh

```diff
@@ -0,0 +1,101 @@
+#!/usr/bin/env bash
```

Comment:
Since you introduces the integration test, can we remove these e2e tests?

Reply:
At first I didn't add them just went for integration test instead, but I understood from some meeting that we still need these so I added one. Not sure if that still holds up, but if it doesn't we could just remove it.